src/test/java/hudson/plugins/emailext/plugins/content/FailedTestsContentTest.java

```diff
@@ -273,4 +274,34 @@ public void testGetContent_withMessage_withStack_htmlEscaped() {
             "Stack Trace:<br/>" +
             "at org.nexusformat.NexusFile.&lt;clinit&gt;(NexusFile.java:99)<br/><br/>");
     }
+
+    @Test
+    public void testGetContent_withMessage_withStack_outputYaml() {
+        AbstractTestResultAction<?> testResults = mock( AbstractTestResultAction.class );
+        when( testResults.getFailCount() ).thenReturn( 1 );
+
+        TestResult result = mock( TestResult.class );
+        when( result.isPassed() ).thenReturn( false );
+        when( result.getFullName() ).thenReturn( "hudson.plugins.emailext.ExtendedEmailPublisherTest" );
+        when( result.getDisplayName() ).thenReturn( "Test" );
+        when( result.getErrorDetails() ).thenReturn( "expected:<ABORTED> but was:<COMPLETED> " );
+        when( result.getErrorStackTrace() ).thenReturn( "at org.nexusformat.NexusFile.<clinit>(NexusFile.java:99)" );
```

Comment:
Right now if adding a test case exceeds the `maxLength`, it will continue to add it, but the tests after it will get truncated. So no `FailedTest`'s yaml will ever get truncated abruptly.

Reply:
I am talking about the yaml encoder splitting long lines, not about a test getting split. I am referring to any possible loss of readability and I will get a clarity once you add a real looking stack trace.

> Right now if adding a test case exceeds the maxLength, it will continue to add it

That is an acceptable behavior.

Reply:
Oh okay. I got it now. I will include a case with proper stacktrace in the unit tests.